gym/lib/gym/runner.rb

```diff
@@ -72,6 +73,11 @@ def clear_old_files
       end
     end
 
+    def fix_archive
+      return unless Gym.config[:use_legacy_build_api]
+      Gym::XcodebuildFixes.generic_archive_fix
```

Comment:
Not yet until we figure out what causes it. I'll put it behind a flag and in the future if we figure out what causes it I can remove the flag and detect it.